backend/geonature/core/errors/routes.py

```diff
@@ -1,6 +1,9 @@
 import logging
+from urllib.parse import urlparse
 
-from flask import current_app, jsonify, Response
+from flask import current_app, jsonify, Response, request, json, redirect
+from werkzeug.exceptions import Unauthorized, HTTPException
+from werkzeug import url_encode
```

Comment:
Sur quelle version de werkzeug tu es ? 
J'ai une erreur d'import avec la 1.0.1
```suggestion
from werkzeug.urls import url_encode
```